art/attacks/evasion/projected_gradient_descent/projected_gradient_descent.py

```diff
@@ -115,6 +122,7 @@ def __init__(
         self.batch_size = batch_size
         self.random_eps = random_eps
         self.verbose = verbose
+        self.tensor_board = tensor_board
```

Comment:
Are there any reason why not to do this in the super().__init__ function?

Reply:
You are right.